package.json

```diff
@@ -9,7 +9,8 @@
   },
   "scripts": {
     "postinstall": "patch-package",
-    "start": "yarn build-styling && multiview [ yarn workspace @dso-toolkit/core start --prod ] [ yarn workspace dso-toolkit start ]",
+    "start": "yarn build-styling && multiview [ yarn workspace @dso-toolkit/core start ] [ yarn workspace dso-toolkit start ]",
```

Comment:
`--prod` is de default, wordt in het nieuwe script `storybook` overruled met `--dev`